internal/log/level.go

```diff
@@ -94,17 +92,17 @@ func ParseLevel(s string) (level Level, err error) {
 	}
 
 	switch strings.ToUpper(s) {
-	case Trace.String(), "TRACE":
+	case Trace.String():
 		return Trace, nil
-	case Debug.String(), "DEBUG":
+	case Debug.String():
 		return Debug, nil
 	case Info.String():
 		return Info, nil
 	case Warn.String():
 		return Warn, nil
-	case Error.String(), "ERROR":
+	case Error.String():
 		return Error, nil
-	case Critical.String(), "CRITICAL":
+	case Critical.String(), "CRIT":
```

Comment:
Let's remove support for `CRIT`? I don't think we use it anywhere, and we can just update flag we use to use `critical` instead of `crit`

```suggestion
	case Critical.String():
```

Reply:
yeah removing it broke tests and I wasn't sure why, imagine its the flag then will go look